filestorage.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const submodules = [
+  'filestorage',
+].map(path => require('./lib/' + path));
+
+module.exports = Object.assign({}, ...submodules);
```

Comment:
I assume this will also expose `mkdir`, that's not a good idea.
Why don't you just
```js
const { Filestorage } = require('./lib/filestorage');

module.exports = {
  Filestorage,
};
```
?

Reply:
Then why do we even need to export `mkdirp`? If there will be other modules that will use it, I'd prefer to put internal functions into a separate file. As for now, I think it shouldn't be exported from the file.

Reply:
Yeah, I don't think it should be exported at all.